ui/src/admin/components/DatabaseRow.js

```diff
@@ -46,51 +48,55 @@ class DatabaseRow extends Component {
         <tr>
           <td>
             {isNew
-              ? <div className="admin-table--edit-cell">
-                  <input
-                    className="form-control"
-                    type="text"
-                    defaultValue={name}
-                    placeholder="Name this RP"
-                    onKeyDown={e => this.handleKeyDown(e, database)}
-                    ref={r => (this.name = r)}
-                    autoFocus={true}
-                  />
-                </div>
-              : <div className="admin-table--edit-cell">
-                  {name}
-                </div>}
-          </td>
-          <td>
-            <div className="admin-table--edit-cell">
-              <input
-                className="form-control"
-                name="name"
-                type="text"
-                defaultValue={formattedDuration}
-                placeholder="How long should Data last"
-                onKeyDown={e => this.handleKeyDown(e, database)}
-                ref={r => (this.duration = r)}
-                autoFocus={!isNew}
-              />
-            </div>
+              ? <input
```

Comment:
Intent was to reduce super specific style use. I think from a functionality standpoint this component is very tied to the Admin UI, so I wouldn't move it. When I was going through the admin styles I realized there's a lot of stuff that was either a) an excessive way to implement the design, or b) something that should become a generic table style instead of being scoped to admin tables.

Previously `.admin-table` carried a lot more weight, whereas now it primarily exists to style  multi-select dropdowns in such a way that they look like plaintext before hover. I have some other styles using the same namespace so I'll keep it around for now, long term my goal is to reduce component specific styles as much as possible